app/src/sharedTest/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -279,79 +284,134 @@ class RevisionCardFragmentTest {
   }
 
   @Test
-  fun testRevisionCardTestActivity_fractionSubtopicId1_checkReturnToTopicButtonIsDisplayedSuccessfully() { // ktlint-disable max-line-length
+  fun testRevisionCardTestActivity_fractionSubtopicId0_checkPreviousNavCardIsNotDisplayed() {
     launch<RevisionCardActivity>(
       createRevisionCardActivityIntent(
         context,
         profileId.internalId,
         FRACTIONS_TOPIC_ID,
-        SUBTOPIC_TOPIC_ID
+        SUBTOPIC_TOPIC_ID_0
       )
     ).use {
       testCoroutineDispatchers.runCurrent()
 
-      onView(withId(R.id.revision_card_return_button))
-        .check(matches(withText(R.string.return_to_topic)))
+      onView(withId(R.id.previous_navigation_card))
+        .check(matches(not((isDisplayed()))))
     }
   }
 
   @Test
-  fun testRevisionCardTestActivity_configurationChange_toolbarTitle_fractionSubtopicId1_isDisplayedCorrectly() { // ktlint-disable max-line-length
+  fun testRevisionCardTestActivity_fractionSubtopicId1_checkPreviousAndNextNavCardsDisplayed() {
     launch<RevisionCardActivity>(
       createRevisionCardActivityIntent(
         context,
         profileId.internalId,
         FRACTIONS_TOPIC_ID,
-        SUBTOPIC_TOPIC_ID
+        SUBTOPIC_TOPIC_ID_1
       )
     ).use {
       testCoroutineDispatchers.runCurrent()
 
-      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.previous_navigation_card))
+        .check(matches(isDisplayed()))
+
+      onView(withId(R.id.next_navigation_card))
+        .check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testRevisionCardTestActivity_fractionSubtopicId3_checkNextNavCardIsNotDisplayed() {
+    launch<RevisionCardActivity>(
+      createRevisionCardActivityIntent(
+        context,
+        profileId.internalId,
+        FRACTIONS_TOPIC_ID,
+        SUBTOPIC_TOPIC_ID_3
+      )
+    ).use {
       testCoroutineDispatchers.runCurrent()
 
-      onView(withId(R.id.revision_card_toolbar_title))
-        .check(matches(withText("What is a Fraction?")))
+      onView(withId(R.id.next_navigation_card))
+        .check(matches(not(isDisplayed())))
     }
   }
 
   @Test
-  fun testRevisionCardTestActivity_configurationChange_fractionSubtopicId2_checkExplanationAreDisplayedSuccessfully() { // ktlint-disable max-line-length
+  fun testRevisionCardTestActivity_fracSubtopicId1_clickPrevNavCard_opensRevisionCardActivity() {
     launch<RevisionCardActivity>(
       createRevisionCardActivityIntent(
         context,
         profileId.internalId,
         FRACTIONS_TOPIC_ID,
-        SUBTOPIC_TOPIC_ID_2
+        SUBTOPIC_TOPIC_ID_1
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+
+      onView(withId(R.id.previous_navigation_card)).perform(click())
+      testCoroutineDispatchers.runCurrent()
+      intended(hasComponent(RevisionCardActivity::class.java.name))
+      intended(hasExtra(SUBTOPIC_ID_EXTRA_KEY, SUBTOPIC_TOPIC_ID_1 - 1))
```

Comment:
For this & the next test, suggest using the direct subtopic ID constant rather than using math (to make it more clear to the reader) _which_ card is being opened.

Reply:
done